### PR TITLE
Fix w_module example

### PR DESCRIPTION
**Pull request by noahbetzen-wk**
## Motivation
  
The example README referenced `pub run dart_dev examples`, but that no longer works. Instead you have to use `pub serve example/`.

Once run, navigating to `localhost:8080/panel/` causes the error pictured below, preventing the entire page from loading.

Screen Shot 2019-04-26 at 2 06 19 PM

## Changes
  
I updated the example directory README.

I don't know enough about the original issue with the `/panel/` page, so I commented out the lines that add the tracer, and now the example works.

#### Release Notes
  
N/A?

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/w_module/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/w_module/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: example/panel/panel_app.dart [hunk 1]
```diff
@@ -31,9 +31,9 @@ Future<Null> main() async {
   Element container = querySelector('#panel-container');
   react_client.setClientConfiguration();
 
-  final tracer = new SampleTracer();
-  initGlobalTracer(tracer);
-  assert(globalTracer() == tracer);
+//  final tracer = new SampleTracer();
+//  initGlobalTracer(tracer);
+//  assert(globalTracer() == tracer);
```
Comment (georgelesica-wf) [inline example/panel/panel_app.dart]: ???

Comment (noahbetzen-wk) [reply example/panel/panel_app.dart]: The only way to get the example to run properly is to comment those out. I assume someone more knowledgable than me will come in and actually fix it, but the `SampleTracer` is the culprit. See the original screenshot I posted.

**Comment by noahbetzen-wk** [comment]
Closing because stale.

**Pull request closed without merging**